lib/libc/gen/Symbol.map

```diff
@@ -210,6 +210,7 @@ FBSD_1.0 {
 	posix_madvise;
 	popen;
 	pclose;
+	psiginfo;
```

Comment:
The symbol should have FBSD_1.8 version.

Reply:
Fixed.